test/tasks/utils.js

```diff
@@ -3,65 +3,33 @@ const DeployTask = require("../../src/worker/tasks/DeployTask");
 const Result = require("../../src/worker/tasks/Result");
 const testUtils = require("../testUtils/utils");
 const constants = require("../../src/common/constants");
+const deployTasks = require("./random_deploy_tasks.json");
+const computeTasks = require("./random_compute_tasks.json");
 
-function generateComputeTasks(num) {
-  let tasks = [];
+function generateTasks(num, type) {
+  const tasks = [];
+  const taskType = type === "deploy" ? DeployTask : ComputeTask;
+  const taskList = type === "deploy" ? deployTasks : computeTasks;
   for (let i = 0; i < num; i++) {
-    let task = ComputeTask.buildTask({
-      userEthAddr: "0x" + testUtils.randLenStr(40),
-      userNonce: testUtils.getRandomInt(100),
-      // H(userEthAddr|userNonce)
-      taskId: "0x" + testUtils.randLenStr(64),
-      encryptedArgs: testUtils.randLenStr(200),
-      encryptedFn: testUtils.randLenStr(200),
-      userDHKey: testUtils.randLenStr(130),
-      contractAddress: "0x" + testUtils.randLenStr(40),
-      gasLimit: testUtils.getRandomInt(100),
-      blockNumber: testUtils.getRandomInt(100)
-    });
-    if (task) tasks.push(task);
-    else console.log("task is null error in generating test data!!!!");
+    const taskJson = taskType.fromDbJson(taskList[i]);
+    taskJson._taskId = testUtils.randLenStr(64);
```

Comment:
Instead of setting the `_taskId` field after the creation of the object, it is better to add the parameter into the dict prior to calling the `fromDbJson`:
```
let taskJson = taskList[i];
taskJson. taskId = testUtils.randLenStr(64);
const task = taskType.fromDbJson(taskJson);
tasks.push(task);
```